api/v1/topologymap.go

```diff
@@ -17,9 +17,12 @@ limitations under the License.
 package v1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	"strings"
 )
 
+const labelZoneFailureDomainWithoutBeta = "failure-domain.kubernetes.io/zone"
```

Comment:
Wasn't this already defined somewhere in the codebase?

Reply:
The variable is defined in another package within the project, but it is a private variable (starts with a lowercase letter). Since it cannot be accessed outside its package.

Reply:
I saw our docs & went as far back as [OCS 4.7.](https://red-hat-storage.github.io/ocs-training/training/ocs4/ocs4-metro-stretched.html) And I can confirm 2 things.
1. Stretch cluster always uses failure domain zone
2. For zone failure domain always `topology.kubernetes.io/zone` label is used. I don't see any need to search for any other labels such as `failure-domain.kubernetes.io/zone`.

Reply:
Agreed with Malay on all these points. `failure-domain.kubernetes.io/zone` is an old deprecated label for topology that was removed from K8s many releases ago, we should not use it anymore.